src/controllers/users.js

```diff
@@ -1,28 +1,42 @@
 import * as Users from "../models/user.js"
+import bcrypt from "bcryptjs"
+import express from "express"
 
 /**
  * Controller for: GET /users/:id
- * @param {Request} req The Request object
- * @param {Response} res The Response object
+ * @param {express.Request} req The Request object
```

Comment:
Accidentally kept these types in when experimenting with @ts-check. 
Not necessary to keep